charts/linkerd-base/templates/identity-rbac.yaml

```diff
@@ -1,4 +1,3 @@
-{{if .Values.identity -}}
```

Comment:
The `linkerd-base` chart's values.yaml file doesn't have any entries for identity, it's all handled by the `linkerd-control-plane` chart's values.yaml. Having this would imply introducing a new `identity` entry in the former. And given I can't think of any scenario where people would want to disable identity, and not even sure if linkerd would work at all, I decided to remove this option.